onboarding/tests/factories.py

```diff
@@ -28,7 +28,7 @@ class Meta:
 
     has_called_311 = False
 
-    lease_type = False
```

Comment:
Uhh, that's super weird that this used to be a boolean, and that tests still worked with it that way. Oh well.